honeybee_radiance_command/gendaylit.py

```diff
@@ -0,0 +1,255 @@
+
+# coding: utf-8
+
+"""Gendaylit command."""
+
+from .options.gendaylit import GendaylitOptions
+from ._command import Command
+from ._typing import tuple_with_length, int_in_range
+import honeybee_radiance_command._exception as exceptions
+
+
+class Gendaylit(Command):
```

Comment:
What if we subclass `gendaylit` from `gensky`? We only need to overwrite the options and the validation. Right?

Reply:
I did that. The tests fail as expected since the Gensky PR is not merged yet.